app/src/main/java/com/globant/harrypotterapp/util/Event.kt

```diff
@@ -0,0 +1,5 @@
+package com.globant.harrypotterapp.util
+
+open class Event<out T>(private val content: T) {
+    fun peekContent(): T = content
+}
```

Comment:
which is the use of this class?

Reply:
This class is used to get the data just one time. But this is not the way to use it. Refactored and thanks by helping me Nico!